### PR TITLE
Fix: fit ```Eulers_totient_function.cpp``` to guidelines

### PR DESCRIPTION
**Pull request by ewd00010**
#### Description of Change
chore: changed some documentation and comments
chore: cleaned code mainly with clang format, clang tidy

#### Checklist

- [x] Added description of change
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: TODO: 

- [x] Change @description to @details in the first part of the documentation
- [x] figure out why i need ```#include <bits/stdint-uintn.h>  // for uint64_t``` in this file and some others

### CODE REVIEW

File: math/eulers_totient_function.cpp [hunk 1]
```diff
@@ -11,7 +13,7 @@
  * 2. \f$\phi(n^k) = n^k - n^{k-1}\f$
  * 3. \f$\phi(a,b) = \phi(a)\cdot\phi(b)\f$ where a and b are relative primes.
  *
- * Applying this 3 properties on the first equation.
+ * Applying the three properties on the first equation.
```
Comment (realstealthninja) [inline math/eulers_totient_function.cpp:16]:
```suggestion
 * Applying these three properties on the first equation.
```

File: math/eulers_totient_function.cpp [hunk 2]
```diff
@@ -63,16 +74,14 @@ static void test() {
...
 /**
  * @brief Main function
- * @param argc commandline argument count (ignored)
- * @param argv commandline array of arguments (ignored)
  * @returns 0 on exit
  */
-int main(int argc, char *argv[]) {
-    test();
+int main() {
+    tests();
```
Comment (realstealthninja) [inline math/eulers_totient_function.cpp:85]:
```suggestion
    tests(); // run self-test implementations
```

File: math/eulers_totient_function.cpp [hunk 3]
```diff
@@ -35,16 +39,23 @@ namespace math {
 /**
  * @brief Function to calculate Euler's Totient
  * @param n the number to find the Euler's Totient of
+ * @return result the result of the claculation
```
Comment (realstealthninja) [inline math/eulers_totient_function.cpp:42]:
```suggestion
 * @returns the result of Euler's Totient of n
```

**Review by realstealthninja — CHANGES_REQUESTED** [review]

File: math/eratosthenes.cpp [hunk 4]
```diff
@@ -1,118 +0,0 @@
-/**
- * @file
- * @brief [The Sieve of
- * Eratosthenes](https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes)
```
Comment (realstealthninja) [inline math/eratosthenes.cpp:4]: why was this deleted

**Review by realstealthninja — CHANGES_REQUESTED** [review]

File: math/eratosthenes.cpp (hunk 4 above)
Comment (ewd00010) [reply math/eratosthenes.cpp:4]: it doesn't look deleted to me, just has the addition of a short description of what the algorithm does right? Well, the file shows it as deleted but then re-added with the addition.

Comment (realstealthninja) [reply math/eratosthenes.cpp:4]: both of these files have different file names and are not the same algoritms

**Review by realstealthninja — CHANGES_REQUESTED** [review]
